nym-vpn-core/nym-vpn-lib/src/platform/swift.rs

```diff
@@ -3,10 +3,11 @@
 
 use log::LevelFilter;
 use oslog::OsLogger;
+use std::str::FromStr;
 
-pub fn init_logs() {
+pub fn init_logs(level: Option<String>) {
     let result = OsLogger::new("net.nymtech.vpn.agent")
-        .level_filter(LevelFilter::Debug)
+        .level_filter(LevelFilter::from_str(&level.unwrap()).unwrap_or(LevelFilter::Info))
```

Comment:
`level.unwrap()` will crash when `level` is `None`.


I think that environment variable overrides the `level_filter`,  i.e`RUST_LOG=debug`.